output_posix.go

```diff
@@ -233,90 +233,120 @@ func (w *PosixWriter) SetTitle(title string) {
 		titleBytes = bytes.Replace(titleBytes, patterns[i].from, patterns[i].to, -1)
 	}
 
-	w.WriteRaw([]byte{0x1b, 0x5d, 0x32, 0x3b})
+	w.WriteRaw([]byte{0x1b, ']', '2', ';'})
 	w.WriteRaw(titleBytes)
 	w.WriteRaw([]byte{0x07})
 	return
 }
 
 // ClearTitle clears a title of terminal window.
 func (w *PosixWriter) ClearTitle() {
-	w.WriteRaw([]byte{0x1b, 0x5d, 0x32, 0x3b, 0x07})
+	w.WriteRaw([]byte{0x1b, ']', '2', ';', 0x07})
 	return
 }
 
 /* Font */
 
 // SetColor sets text and background colors. and specify whether text is bold.
+// Deprecated. This interface is not cool, please use SetDisplayAttributes.
 func (w *PosixWriter) SetColor(fg, bg Color, bold bool) {
+	if bold {
+		w.SetDisplayAttributes(fg, bg, DisplayBold)
+	} else {
+		w.SetDisplayAttributes(fg, bg, DisplayDefaultFont)
+	}
+	return
+}
+
+// SetDisplayAttributes set display attributes (Set colors, blink, bold, italic and so on).
+func (w *PosixWriter) SetDisplayAttributes(fg, bg Color, attrs ...DisplayAttribute) {
```

Comment:
Add SetDisplayAttributes for flexible customizing.
I need to update ConsoleWriter interface.